actionpack/lib/action_dispatch/middleware/ssl.rb

```diff
@@ -123,7 +123,13 @@ def flag_cookies_as_secure!(headers)
       end
 
       def redirect_to_https(request)
-        [ @redirect.fetch(:status, redirection_status(request)),
+        status = @redirect.fetch(:status, nil)
```

Comment:
```suggestion
        status = @redirect[:status]
```
IIRC, `@redirect` is a hash-like object, defaulting to nil if not found,

Reply:
Good point :+1: I've fixed it.